docs/GettingStartedDocs/Contributors/WindowsManualInstallPrereqs.md

```diff
@@ -11,19 +11,21 @@
    - To check your Windows version, run `winver` on the command line.
 
 ## Software prerequisites
+For specific versions we use for each release, see [install-windows-prereqs.ps1](https://github.com/openenclave/openenclave/blob/master/scripts/install-windows-prereqs.ps1#L6-L36).
 - [Microsoft Visual Studio Build Tools 2019](https://aka.ms/vs/15/release/vs_buildtools.exe)
 - [Git for Windows 64-bit](https://git-scm.com/download/win)
-- [Clang/LLVM for Windows 64-bit](https://github.com/llvm/llvm-project/releases/download/llvmorg-8.0.1/LLVM-8.0.1-win64.exe)
+- [Clang/LLVM 8.0.1 for Windows 64-bit](https://github.com/llvm/llvm-project/releases/download/llvmorg-8.0.1/LLVM-8.0.1-win64.exe)
 - [Python 3](https://www.python.org/downloads/windows/)
-- [ShellCheck](https://oejenkins.blob.core.windows.net/oejenkins/shellcheck-v0.7.0.zip)
-- [OpenSSL 1.1.1](https://oejenkins.blob.core.windows.net/oejenkins/openssl.1.1.1506.73.nupkg)
+- [ShellCheck](https://github.com/koalaman/shellcheck/releases)
+- [OpenSSL 1.1.1](https://wiki.openssl.org/index.php/Binaries)
```

Comment:
@anakrish, should we keep the openssl linked here earlier? My understanding is that you and Brett worked together so we need not download the windows binaries here: https://wiki.openssl.org/index.php/Binaries

Reply:
Please keep lines 81-83 in context as well, as I didn't want to link to a old version if OpenSSL is updated.
> Download and install the latest build of OpenSSL for Windows from one of the following [locations](https://wiki.openssl.org/index.php/Binaries) or build and install it from source.
Alternatively, you can use the latest version of OpenSSL that the OpenEnclave CI team [publishes](https://oejenkins.blob.core.windows.net/oejenkins/openssl.1.1.1506.73.nupkg) which is distributed under the [dual OpenSSL and SSLeay license](https://www.openssl.org/source/license-openssl-ssleay.txt) without further legal obligations.

Reply:
OK, I am fine with this. Thanks.

Reply:
It is preferable to point to the openssl binary that we publish, instead of the ones pointed to by OpenSSL wiki. 
I'd suggest that we change the wording in lines 81-83 to reflect this.
It'd be good if we had a link `https://oejenkins.blob.core.windows.net/oejenkins/openssl.latest.nupkg` that always points to the latest version we support.

Reply:
I like this idea. I'll create the new package and have it point to that. Thanks!

Reply:
Added this in https://github.com/openenclave/openenclave/pull/4168/files#diff-a9aa2de6b0535683147f83afc0f48e8b442218d23a05c236189ee781b6169142